.travis.yml

```diff
@@ -4,21 +4,20 @@ os:
   - osx
 julia:
   - 0.4
+  - 0.5
 notifications:
   email: false
 addons:
   apt:
     packages:
     - cmake
     - libxxf86vm-dev
+    - libxinerama-dev
+    - libxcursor-dev
     sources:
     - kubuntu-backports
+before_script:
+  - if [ "${TRAVIS_OS_NAME}" = "linux" ]; then export DISPLAY=:99.0; sh -e /etc/init.d/xvfb start; fi
 script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia -e 'Pkg.clone(pwd()); Pkg.checkout("GLAbstraction", "sd/next²")'
-  - julia -e 'Pkg.checkout("GeometryTypes", "sd/typealias")'
-  - julia -e 'Pkg.checkout("GLWindow");Pkg.checkout("Reactive")'
-  - julia -e 'Pkg.checkout("Packing");Pkg.checkout("FreeType")'
-  - julia -e 'Pkg.test("GLVisualize"; coverage=true)'
-after_success:
-    - julia -e 'cd(Pkg.dir("GLVisualize")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
+  - julia -e 'Pkg.clone(pwd()); using GLVisualize'
```

Comment:
No :( But the next release cycle should address decoupling GLVisualize from OpenGL and headless rendering, which hopefully finally resolves this!